pkg/plugins/optional/helm/v1alpha/scaffolds/init.go

```diff
@@ -560,3 +571,103 @@ func hasWebhooksWith(c config.Config) bool {
 
 	return false
 }
+
+// readManagerSpecs reads the manager.yaml file from config/manager/manager.yaml
+// and extracts relevant specs to be used in values.yaml generation
+func (s *initScaffolder) readManagerSpecs() error {
```

Comment:
Would it be feasible to generate the manager from configuration files, instead of maintaining boilerplate code for both the manager and the Helm plugin? Could this approach work effectively, or are there potential pitfalls we should be aware of?